pkgs/top-level/all-packages.nix

```diff
@@ -24250,15 +24241,18 @@ with pkgs;
 
   vulkan-cts = callPackage ../tools/graphics/vulkan-cts { };
 
-  vulkan-extension-layer = callPackage ../tools/graphics/vulkan-extension-layer { };
-  vulkan-headers = callPackage ../development/libraries/vulkan-headers { };
-  vulkan-loader = callPackage ../development/libraries/vulkan-loader { inherit (darwin) moltenvk; };
-  vulkan-tools = callPackage ../tools/graphics/vulkan-tools {
-    inherit (darwin) moltenvk;
-    inherit (darwin.apple_sdk.frameworks) AppKit Cocoa;
-  };
-  vulkan-tools-lunarg = callPackage ../tools/graphics/vulkan-tools-lunarg { };
-  vulkan-validation-layers = callPackage ../development/tools/vulkan-validation-layers { };
+  vulkanPackages_sdk = recurseIntoAttrs (import ../development/vulkan {
```

Comment:
In this case, that we have all the packages inherited, should I use `dontRecurseIntoAttrs` instead?